docs/source/tutorials/token_classification/tutorial-p2.ipynb

```diff
@@ -0,0 +1,522 @@
+    "# termcolor==1.1.0 "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d0d2e007",
+   "metadata": {},
+   "source": [
+    "# Token Classification Label Error Detection - Part 2 \n",
+    "\n",
+    "In this tutorial, we show how you can use cleanlab to find potential label errors in a NLP token classification dataset. Here, we use CONLL-2003 which contains 20,718 sentences and 301,361 tokens. In a standard NLP token-classification task, the model predicts the entity of each token within each sentence. In this example, the named entity includes miscellaneous `MISC`, location `LOC`, person `PER`, and organization `ORG`. Unnamed entities are labeled as `O`. \n",
+    "\n",
+    "**Overview of what we'll do in this tutorial:** \n",
+    "- Identify potential token label issues using cleanlab's `token_classification.filter.find_label_issues` method. \n",
+    "- Rank sentences using cleanlab's `token_classification.rank.get_label_quality_score` method. \n",
+    "- TODO: (Clean Learning) Train a more robust model by removing problematic sentences. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1da020bc",
+   "metadata": {},
+   "source": [
+    "## 1. Install the required dependencies "
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "a1349304",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import numpy as np\n",
+    "from termcolor import colored \n",
+    "from cleanlab.token_classification.filter import find_label_issues \n",
+    "from cleanlab.token_classification.rank import get_label_quality_scores\n",
+    "from utils import * \n",
+    "# change to this after token_classification.rank is pushed to the official package \n",
+    "# from cleanlab.token_classification.rank import get_label_quality_scores "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "9ad75b45",
+   "metadata": {},
+   "source": [
+    "## 2. Get `pred_probs` and `labels` \n",
+    "\n",
+    "`pred_probs` are out-of-sample model-predicted probabilities of the CoNLL-2003 dataset (including training, development, and testing dataset), obtained via cross-validation. To detect potential labels issues, we first get `pred_probs` and `labels`, which are both in nested-list format, such that: \n",
+    "\n",
+    "- `pred_probs` is a list of `np.arrays`, such that `pred_probs[i]` is the model-predicted probabilities for the tokens in the i'th sentence, and has shape `(N_i, K)`, where `N_i` is the number of word-level tokens of the `i`'th sentence. Each row of the matrix corresponds to a token `t` and contains the model-predicted probabilities that `t` belongs to each possible class, for each of the K classes. The columns must be ordered such that the probabilities correspond to class 0, 1, ..., K-1. \n",
+    "        \n",
+    "- `labels` is a list of lists, such that `labels[i]` is a list of token labels of the `i`'th sentence. Same format requirements as `cleanlab.rank.get_label_quality_scores`. \n",
```

Comment:
I'd delete this sentence, since the format is a bit different given `labels[i]` here is a vector, whereas it is a scalar in `cleanlab.rank.get_label_quality_scores`

Reply:
Would instead just explicitly explain what `labels` should like